standard/classes.md

```diff
@@ -1565,7 +1565,7 @@ A constant declaration that declares multiple constants is equivalent to multipl
 >
 > *end example*
 
-Constants are permitted to depend on other constants within the same program as long as the dependencies are not of a circular nature. The compiler automatically arranges to evaluate the constant declarations in the appropriate order.
+Constants are permitted to depend on other constants within the same program as long as the dependencies are not of a circular nature. A compiler automatically arranges to evaluate the constant declarations in the appropriate order.
```

Comment:
This second sentence doesn't sound quite like spec-ese to start with. Presumably it should be more of a *requirement* instead of a statement of what happens.

Reply:
reworked the sentence to permit a compiler to reorder the evaluation of constant declarations